po/R-dk.po

```diff
@@ -0,0 +1,287 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: pkgdown 2.0.1.9000\n"
+"POT-Creation-Date: 2021-12-20 18:48+0100\n"
+"PO-Revision-Date: 2021-12-20 18:48+0100\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: none\n"
+"Language: dk\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: build-404.R:11
+msgid "Page not found (404)"
+msgstr "Side ikke fundet"
+
+#: build-404.R:12
+msgid "Content not found. Please use links in the navbar."
+msgstr "Indhold ikke fundet. Brug venligst links i menuen"
+
+#: build-articles.R:379 navbar.R:134 navbar.R:141 navbar.R:156
+msgid "Articles"
+msgstr "Artikler"
+
+#: build-articles.R:428
+msgid "All vignettes"
+msgstr "Alle vignetter"
+
+#: build-footer.R:29
+msgid "Developed by"
+msgstr "Udviklet af"
+
+#: build-footer.R:34
+msgid "Site built with <a href=\"%s\">pkgdown</a> %s."
+msgstr "Hjemmeside bygget med <a href=\"%s\">pkgdown</a> %s."
```

Comment:
"Hjemmeside" sounds a bit '90s 🙂. Maybe just "Side"? I have no strong feelings about this however

Reply:
Ha, ha! I totally agree 😃